blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/serializers/NumericSerializer.java

```diff
@@ -535,7 +556,7 @@ private static AbstractRollupStat getStatFromRollup(byte statType, BasicRollup b
         public ByteBuffer toByteBuffer(Object o) {
             try {
                 byte type = typeOf(o);
-                byte[] buf = new byte[sizeOf(o, type)];
+                byte[] buf = new byte[sizeOf(o, type, VERSION_2_TIMER)];
```

Comment:
It seems a bit strange to pass in the `timer` version while serializing non-timer rollups like gauges, counters etc. What do you think about having two overloaded methods named `sizeOf` with one accepting just two arguments i.e `(Object, Type)` which will be used by all non-timer rollup types and then the other one with `timerVersion` as the third argument, which gets called by the timer serializer?